cypress/page/upload.page.ts

```diff
@@ -0,0 +1,31 @@
+
+class Upload {
+    private URL: string;
+    private selectFileSelector: string;
+    private uploadFileSelector: string;
+    private titleNameSelector: string;
+
+    constructor(){
+        this.URL = "https://the-internet.herokuapp.com/upload"
+        this.selectFileSelector = "#file-upload"
+        this.uploadFileSelector = "#file-submit"
+        this.titleNameSelector = "#uploaded-files"
+    }
+
+    public visitWebSite(): void{
+        cy.visit(this.URL)
+    }
+
+    public uploadFile(fileName: string): void{
+        cy.get(this.selectFileSelector).attachFile(fileName)
+        cy.get(this.uploadFileSelector).click()
+    }
+
+    public verifyTitle(data: string[]): void {
+        cy.get(this.titleNameSelector).each((item, index) => {
+            cy.wrap(item).should("contain.text", data[index])
+          })
```

Comment:
this can be as simple as:
cy.get(this.titleNameSelector).should("have.text", fileName) 
and the fileName be simply a string and not an array